packages/flex-dev-utils/src/packages/lib/__tests__/getLatestFlexUIVersion.test.ts

```diff
@@ -0,0 +1,109 @@
+import * as getRegistryVersionScripts from '../getRegistryVersion';
+import getLatestFlexUIVersion from '../getLatestFlexUIVersion';
+
+describe('getLatestFlexUIVersion', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    jest.restoreAllMocks();
+  });
+
+  it('should return the latest verison', async () => {
+    const getRegistryVersion = jest.spyOn(getRegistryVersionScripts, 'default');
+    // @ts-ignore
+    getRegistryVersion.mockResolvedValueOnce({ version: '2.0.0' });
+
+    await getLatestFlexUIVersion(2);
+
+    expect(getRegistryVersion).toHaveBeenCalledTimes(1);
+    expect(getRegistryVersion).toHaveBeenCalledWith('@twilio/flex-ui', 'latest');
```

Comment:
make `@twilio/flex-ui` a const here as it is being used multiple times throughout the test